src/data/polynomial/basic.lean

```diff
@@ -104,6 +104,13 @@ show neg _ = _, by rw neg
 lemma mul_to_finsupp {a b} : (⟨a⟩ * ⟨b⟩ : polynomial R) = ⟨a * b⟩ := show mul _ _ = _, by rw mul
 lemma smul_to_finsupp {S : Type*} [monoid S] [distrib_mul_action S R] {a : S} {b} :
   (a • ⟨b⟩ : polynomial R) = ⟨a • b⟩ := rfl
+lemma is_smul_regular {S : Type*} [monoid S] [distrib_mul_action S R] {a : S}
```

Comment:
Also an extra newline, since the proof is long enough to deserve some space.
```suggestion

lemma _root_.is_smul_regular.polynomial {S : Type*} [monoid S] [distrib_mul_action S R] {a : S}
```